src/py/flwr/common/address.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Adap GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower IP address utils."""
+
+
+from ipaddress import ip_address
+from typing import Tuple, Union
+
+IPV6: int = 6
+
+
+def parse_address(address: str) -> Union[Tuple[str, int, bool], None]:
+    """Parses a string representing an IP address into a host, port and
+    version.
```

Comment:
Shorten to one line

Reply:
@danieljanes This actually breaks the `docformatter` test, should I try to reduce the text so it fits in one line? Like `Parses an IP address into a host, port, and version.`?